pkg/util/labels.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import (
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/pkg/labels"
+)
+
+// LabelsToMetric converts a Labels to Metric
+func LabelsToMetric(ls labels.Labels) model.Metric {
```

Comment:
We have an almost exact copy of this function here: https://github.com/cortexproject/cortex/blob/master/pkg/ingester/client/compat.go#L211, can we dedupe?

Reply:
It feels bad adding a dependency on `pkg/ingester/client` in places that aren't ingester clients.
(although I did that in a few places for `client.Fingerprint()` to avoid moving the FNV code)

However I can easily have `client.FromLabelAdaptersToMetric()` call this one.